pkg/proto/query_coord.proto

```diff
@@ -1012,6 +1014,43 @@ message ValidateAnalyzerRequest{
     repeated AnalyzerInfo analyzer_infos = 2;
 }
 
+// HighlightTask fetch highlight for all queries at one field
+// len(target_texts) == len(topks) == nq
+// len(texts) == sum(topks)
+message HighlightTask{
+    string field_name = 1;
+    int64 field_id = 2;
+    repeated string texts = 3;
+    repeated string analyzer_names = 4;
+
+    int64 search_text_num = 5;
+    int64 corpus_text_num = 6;
```

Comment:
please add comment for `texts`, `analyzer_names`, `search_text_num` and `corpus_text_num`